garminexport/garminbackup.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python
+"""Performs (incremental) backups of activities for a given Garmin Connect
```

Comment:
This comment belongs in the function head of `garminbackup()`

Reply:
moved. but left the original comment block with some cosmetic changes